sys/include/net/ppptype.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2015 José Ignacio Alamos
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @defgroup    net_ppptypes PPP types
+ * @ingroup     net
+ * @brief       PPP types
+
```

Comment:
Link to the database might be helpful for future extensions. See [ethertype.h](https://github.com/RIOT-OS/RIOT/blob/master/sys/include/net/ethertype.h#L13-L15) for reference